cmd/chartsvc/handler.go

```diff
@@ -61,34 +64,110 @@ type rel struct {
 	Links selfLink    `json:"links"`
 }
 
-// listCharts returns a list of charts
-func listCharts(w http.ResponseWriter, req *http.Request) {
+type meta struct {
+	TotalPages int `json:"totalPages"`
+}
+
+// getPageNumberAndSize extracts the page number and size of a request. Default (1, 0) if not set
+func getPageNumberAndSize(req *http.Request) (int, int) {
+	page := req.FormValue("page")
+	size := req.FormValue("size")
+	pageInt, err := strconv.ParseUint(page, 10, 64)
+	if err != nil {
+		pageInt = 1
+	}
+	sizeInt, _ := strconv.ParseUint(size, 10, 64)
```

Comment:
add a comment above here that ParseUint will return 0 if size is not a positive integer